<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>843</width>
    <height>466</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(42, 42, 42);
background-color: rgb(209, 209, 209);</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="btnSTART">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>90</y>
      <width>102</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(55, 29, 255);
selection-color: rgb(0, 85, 255);</string>
    </property>
    <property name="text">
     <string>START/STOP</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnINTRO_MARK">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>90</y>
      <width>106</width>
      <height>30</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(55, 29, 255);
selection-color: rgb(0, 85, 255);</string>
    </property>
    <property name="text">
     <string>INTRO/MARK</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnSTYLUS">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>180</y>
      <width>80</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(55, 29, 255);
selection-color: rgb(0, 85, 255);</string>
    </property>
    <property name="text">
     <string>STYLUS</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnLAMP">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>180</y>
      <width>80</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(55, 29, 255);
selection-color: rgb(0, 85, 255);</string>
    </property>
    <property name="text">
     <string>LAMP</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblSTAR">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>130</y>
      <width>68</width>
      <height>22</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(0, 0, 0);</string>
    </property>
    <property name="text">
     <string>status</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblSTYLUS">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>230</y>
      <width>68</width>
      <height>22</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(0, 0, 0);</string>
    </property>
    <property name="text">
     <string>status</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblLAMP">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>230</y>
      <width>68</width>
      <height>22</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(0, 0, 0);</string>
    </property>
    <property name="text">
     <string>status</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>171</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-color: rgb(145, 145, 145);
font: 75 italic 12pt &quot;PibotoLt&quot;;
color: rgb(0, 0, 0);</string>
    </property>
    <property name="text">
     <string>Machine-Recorder</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="btnLOCK">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>90</y>
      <width>113</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(55, 29, 255);
selection-color: rgb(0, 85, 255);</string>
    </property>
    <property name="text">
     <string>LOCK GROOVE</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnVACUM">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>180</y>
      <width>80</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(55, 29, 255);
selection-color: rgb(0, 85, 255);</string>
    </property>
    <property name="text">
     <string>VACUM</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnFWD">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>180</y>
      <width>80</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(55, 29, 255);
selection-color: rgb(0, 85, 255);</string>
    </property>
    <property name="text">
     <string>&lt;&lt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblFWD">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>230</y>
      <width>68</width>
      <height>22</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(0, 0, 0);</string>
    </property>
    <property name="text">
     <string>status</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblVACUM">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>230</y>
      <width>68</width>
      <height>22</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(0, 0, 0);</string>
    </property>
    <property name="text">
     <string>status</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblBWD">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>130</y>
      <width>68</width>
      <height>22</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(0, 0, 0);</string>
    </property>
    <property name="text">
     <string>status</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnBWD">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>90</y>
      <width>80</width>
      <height>30</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(55, 29, 255);
selection-color: rgb(0, 85, 255);</string>
    </property>
    <property name="text">
     <string>&gt;&gt;</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_2">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>0</y>
      <width>171</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-color: rgb(145, 145, 145);
font: 75 italic 12pt &quot;PibotoLt&quot;;
color: rgb(0, 0, 0);</string>
    </property>
    <property name="text">
     <string>Status:</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="func">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>20</y>
      <width>111</width>
      <height>22</height>
     </rect>
    </property>
    <property name="contextMenuPolicy">
     <enum>Qt::PreventContextMenu</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(0, 0, 0);</string>
    </property>
    <property name="text">
     <string>status</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnHOME">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>90</y>
      <width>113</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(55, 29, 255);
selection-color: rgb(0, 85, 255);</string>
    </property>
    <property name="text">
     <string>HOME</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>843</width>
     <height>27</height>
    </rect>
   </property>
   <property name="contextMenuPolicy">
    <enum>Qt::DefaultContextMenu</enum>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
